{"uuid":"0d89327f-929a-4a30-bb30-da180e382e45","name":"Scenario Outline: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","children":["63d9a732-739f-4580-81f3-57225c75e0a7"],"befores":[{"name":"appHooks.Hooks.scenario(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673381475541,"stop":1673381475542}],"afters":[{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673381475548,"stop":1673381475548},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673381475859,"stop":1673381475859},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673381476046,"stop":1673381476047},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"463abe2b-df7a-4738-a857-21c059f4fad8-attachment","type":"image/png"},{"name":"Myscreenshot","source":"11b6ec8e-6076-461d-8173-13317b5f1747-attachment"}],"parameters":[],"start":1673381476057,"stop":1673381476356}],"start":1673381475541,"stop":1673381476356}